src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -108,21 +107,12 @@ const ActionSidebar: FC<PropsWithChildren<ActionSidebarProps>> = ({
   const actionNotFound = transactionId && !action;
 
   const getSidebarContent = () => {
-    if (isLoading) {
+    if (action || isLoading) {
       return (
-        <div className="flex h-full flex-col items-center justify-center gap-4">
-          <SpinnerLoader appearance={{ size: 'huge' }} />
-          <p className="text-gray-600">
-            {formatText({ id: 'actionSidebar.loading' })}
-          </p>
-        </div>
+        <CompletedAction action={action ?? undefined} isLoading={!!isLoading} />
```

Comment:
Instead of passing props to a component in charge of displaying action data, I think it would be better to combine `ActionContentLoader` and `WidgetContentLoader` into a component called `ActionSidebarLoadingSkeleton` or something, and if it's loading, render it, if it's not, render the `CompletedAction` component.

Thus `CompletedAction` still has just one job (render the appropriate action) and the loading state is handled here

Reply:
@bassgeta done